internal/resolvers/twitter/api.go

```diff
@@ -111,5 +112,8 @@ func getUserByName(userName, bearer string) (*TwitterUserApiResponse, error) {
 		return nil, errors.New("unable to process response")
 	}
 
+	// Do modifications to response
```

Comment:
Either drop this comment or explain why the modification is necessary (could also link to the API documentation here)